src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpRequestCallback.cs

```diff
@@ -0,0 +1,360 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Net.Security;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography.X509Certificates;
+
+using SafeWinHttpHandle = Interop.WinHttp.SafeWinHttpHandle;
```

Comment:
I'm curious why this got introduced in multiple places. I thought we want to keep the `Interop...` part to all types throughout the code.